src/components/app.tsx

```diff
@@ -19,6 +20,7 @@ export const App: React.FC = () => {
   return (
     <ThemeProvider theme={theme}>
       <RecoilRoot>
+        <DebugObserver />
```

Comment:
to remove?

Reply:
open console and run `localStorage.setItem('DEBUG', true)`

Reply:
Got it 👍🏼  Maybe we could expose a global fn, `enableDebug()` in dev mode?